akka-actor-tests/src/test/scala/akka/pattern/RetrySpec.scala

```diff
@@ -0,0 +1,84 @@
+package akka.pattern
+
+import language.postfixOps
+import akka.testkit.AkkaSpec
+
+import scala.concurrent.{ Await, ExecutionContextExecutor, Future }
+import scala.concurrent.duration._
+
+class RetrySpec extends AkkaSpec with RetrySupport {
+  implicit val ec: ExecutionContextExecutor = system.dispatcher
+
+  "pattern.retry" must {
+    "run a successful Future immediately" in {
+      val retried = retry(
+        () ⇒ Future.successful(5),
+        5,
```

Comment:
also check that it was only executed once

Reply:
Fixed